corehq/apps/app_manager/static/app_manager/json/commcare-settings-layout.yml

```diff
@@ -61,6 +61,7 @@
     - hq.translation_strategy
     - hq.mobile_ucr_restore_version
     - hq.location_fixture_restore
+    - hq.split_screen_dynamic_search
```

Comment:
This reminds me we might want to split out a "web apps" section and/or rename the android section to something like "UI", but that's not part of this PR.